backend/src/main/java/com/digginroom/digginroom/repository/MemberRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.digginroom.digginroom.repository;
+
+import com.digginroom.digginroom.domain.Member;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface MemberRepository extends JpaRepository<Member, Long> {
+
+    Optional<Member> findByMemberId(String memberId);
```

Comment:
```suggestion
    Optional<Member> findByMemberId(String memberId);
    
    boolean existsByMemberId(String memberId);
```

해당 `memberId`가 존재하는지 확인하는 메서드를 추가하면 어떨까요?